src/core/scene/a-scene.js

```diff
@@ -320,7 +364,16 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+        // Postproc must be explicitly enabled by a component or system.
```

Comment:
Instead of using `enablePostProcessing` that is specific from the `postprocess` component, just call `this.tock` all the time, if the scene doesn't have any `tock` behaviours it won't execute anything anyway